README.md

```diff
@@ -75,7 +85,9 @@ make
 ### Using Docker
 
 We provide a Debian-based Docker image on Docker Hub with the dependencies
-already installed to make it easier to build Pyodide.
+already installed to make it easier to build Pyodide. Note that building from
+the Docker image is *very* slow on Mac, building on the host machine is
```

Comment:
`on Mac` -> `non-Linux hosts`?

Reply:
Is Docker for Windows actually all that slow?

Reply:
Let's punt on this for now, the wording is ok as-is IMO. At some point in the future maybe a Windows-user can add some more notes for their platform.